WISE-Community/WISE
# Fixed bug where one could not import drawing in public note

Fixes #1358

Steps to verify:
1. As student 1, save a drawing to public space.
2. As student 2, import the drawing from student 1.

The drawing should import without any problems.
